### ISSUE TITLE
Add a UI to display a quote and its author.

### ISSUE THREAD

**Issue by schleepy401** [Ly0kami/QuotesApp-JWoC#1]
Once the splash screen is ready, we will move ahead with a simple issue. **Just build up a sample UI to display a quote and its author, both of which are texts.**
The data source for the quotes would be a part of another issue, but for now, you can just add random quotes from the internet with their authors.

Get your creative juices flowing, and let's see what you people can come up with!
The best looking ones will be merged.

## What to do?
- The data to be displayed is going to be comprised of two parts:
    - **Quote:** A string.
        Example: *"JWoC is Amazing."*
    - *Author:* Another string.
         Example: *Sanskar*
         
- Combining the above two would make something like this:
> *"JWoC is Amazing."* ~ Sanskar

- You have to find a creative way of displaying the above information in a way that looks good and is user friendly as well. Feel free to use any of the components provided by Android Studio.
- The best looking one would be accepted.
- Go ahead, play with components and color schemes!

**Comment by Shreyan-12345** [comment]
do we have to add quotes and author's name in database and then print? @Ly0kami

**Comment by pree-T** [comment]
I am interested to work on this? Kindly assign it to me @Ly0kami

**Comment by shubhanshu-coder** [comment]
can i work on it ?

**Comment by manishbajagai2** [comment]
I want to work on this issue under JWOC

**Comment by shubhanshu-coder** [comment]
i would like to work on it ?

**Comment by ahamedibrahim2002** [comment]
#6 Done this issue (Different Version) 
ahamed | ahmedibrahim2002 (jwoc)

**Comment by anubhab1710** [comment]
Please add **Medium** label instead of Moderate @Ly0kami
